ERCS/erc-7432.md

```diff
@@ -4,7 +4,7 @@ title: Non-Fungible Token Roles
 description: Role Management for NFTs. Enables accounts to share the utility of NFTs via expirable role assignments.
 author: Ernani São Thiago (@ernanirst), Daniel Lima (@karacurt)
 discussions-to: https://ethereum-magicians.org/t/eip-7432-non-fungible-token-roles/15298
-status: Review
+status: Final
```

Comment:
```suggestion
status: Last Call
last-call-deadline: 2024-10-30
```

You must go through Last Call before you can go to Final. Last Call can be as short as 14 days, but because you can't enable edits from maintainers, it's safer to set a further deadline.

Reply:
Hi @SamWilsn 

That's funny, we merged into last call a couple weeks ago. See https://github.com/ethereum/ERCs/pull/609

This was merged to `master`, right? Am I missing something?

Reply:
Hm, could you try `cherry-pick`ing your most recent commit on top of `master`? It seems like you're pushing three commits, at least one of which is already applied.

Reply:
Hi @SamWilsn 

Couldn't remove the conflicting commits, so I created a new branch with the update.

Closing this PR in favor of #655